types/chec__commerce.js/features/products.d.ts

```diff
@@ -1,17 +1,23 @@
 import Commerce = require('@chec/commerce.js');
 import { Product } from '../types/product';
 import { PaginationMeta } from '../types/pagination';
+import { VariantOption } from '../types/variant-option';
 
 export interface ProductCollection {
   data: Product[];
   meta: PaginationMeta;
 }
 
+export interface VariantOptionCollection {
+  data: VariantOption[];
+  meta: PaginationMeta;
+}
+
 export class Products {
   constructor(commerce: Commerce);
 
   list(params?: any): Promise<ProductCollection>;
   retrieve(id: string, data?: object): Promise<Product>;
-  getVariants(id: string, data?: object): Promise<any>;
-  getVariant(id: string, variantId: string): Promise<any>;
+  getVariants(id: string, data?: object): Promise<VariantOptionCollection>;
+  getVariant(id: string, variantId: string): Promise<VariantOption>;
```

Comment:
These return a list of Variant models and a single Variant, not a variant option